consensus/module.go

```diff
@@ -1,263 +0,0 @@
-package consensus
```

Comment:
I think we should keep this file.

Reply:
I had intended to delete the changes not the file hold on